src/pages/MyTransfers/TransfersTable.js

```diff
@@ -0,0 +1,229 @@
+import {
```

Comment:
I would increase the width and height of the table. Please, try 100% for both, and see what it looks like (we should have some space close to screen edges of course and we don't want to have a scroll on the page as well).